src/common/time/src/range.rs

```diff
@@ -18,8 +18,9 @@ use crate::Timestamp;
 
 /// A half-open time range.
 ///
-/// The time range contains all timestamp `ts` that `ts >= start` and `ts < end`. It is
-/// empty if `start >= end`.
+/// The range contains values that `value >= start` and `val < end`.
+///
+/// The range is empty iff `start == end == "the default value of T"`
```

Comment:
typo: iff -> if

Reply:
Looks like when `start >= end`, the range is empty too.

Reply:
> typo: iff -> if

iff means "if and only if".